synapse/push/bulk_push_rule_evaluator.py

```diff
@@ -125,28 +109,43 @@ def __init__(self, hs: "HomeServer"):
         # Whether to support MSC3772 is supported.
         self._relations_match_enabled = self.hs.config.experimental.msc3772_enabled
 
+    async def get_rules(
+        self, room_id: str, event: EventBase
+    ) -> Dict[str, List[Dict[str, dict]]]:
+        """Given an event return the rules for all users who are
+        currently in the room.
```

Comment:
Perhaps: `for all users currently in the room who should be notified of this event`?

Has this event been persisted yet?